### ISSUE TITLE
train network in caffe

### ISSUE THREAD

**Issue by LZJ-Roger** [microsoft/human-pose-estimation.pytorch#128]
I try to train simplebaseline(resnet18) in caffe framework. However, the loss decreases so slowly and the prediction is a little bad.
